src/spring-cloud/azext_spring_cloud/commands.py

```diff
@@ -274,3 +279,12 @@ def load_command_table(self, _):
 
     with self.command_group('spring-cloud', exception_handler=handle_asc_exception):
         pass
+
+    with self.command_group('spring-cloud build-service builder buildpack-binding',
+                            custom_command_type=buildpack_binding_cmd_group,
+                            exception_handler=handle_asc_exception, is_preview=True) as g:
+        g.custom_command('create', 'create_or_update_buildpack_binding')
+        g.custom_command('set', 'create_or_update_buildpack_binding')
+        g.custom_show_command('show', 'buildpack_binding_show')
+        g.custom_command('list', 'buildpack_binding_list')
+        g.custom_command('delete', 'buildpack_binding_delete')
```

Comment:
Do we need consider adding `confirmation=True` for the delete command?

Reply:
make sense, let me update it.

Reply:
updated.